packages/ui/src/components/va-button-group/VaButtonGroup.vue

```diff
@@ -11,35 +11,48 @@ import { defineComponent, computed } from 'vue'
 import { extractComponentProps } from '../../utils/component-options'
 import omit from 'lodash/omit.js'
 
-import { useBem, useComponentPresetProp } from '../../composables'
+import { useBem, useComponentPresetProp, useColors, useTextColor } from '../../composables'
 
 import { VaConfig } from '../va-config'
 import { VaButton } from '../va-button'
 
-const VaButtonProps = omit(extractComponentProps(VaButton), ['block'])
+const VaButtonProps = omit(extractComponentProps(VaButton), ['block', 'gradient'])
 
 export default defineComponent({
   name: 'VaButtonGroup',
   components: { VaConfig },
   props: {
     ...VaButtonProps,
     ...useComponentPresetProp,
-    grow: {
-      type: Boolean,
-      default: false,
-    },
+    grow: { type: Boolean, default: false },
+    gradient: { type: Boolean, default: false },
   },
 
   setup: (props) => {
-    const buttonConfig = computed(() => ({ VaButton: { ...props } }))
+    const { getColor, getGradientBackground } = useColors()
+    const colorComputed = computed(() => getColor(props.color))
+    const { textColorComputed } = useTextColor(colorComputed)
+
+    const buttonConfig = computed(() => ({
+      VaButton: {
+        ...omit(props, 'gradient', 'grow'),
```

Comment:
You can use `filterComponentProps`, or whatever it called, from `'../../utils/component-options'`

Reply:
done